tests/lib/rules/block-scoped-var.js

```diff
@@ -31,6 +31,8 @@ eslintTester.addRuleTest("lib/rules/block-scoped-var", {
         "function f() { var hasOwnProperty; { hasOwnProperty; } }",
         "function f(){ a; b; var a, b; }",
         "function f(){ g(); function g(){} }",
+        { code: "function myFunc(foo) {  \"use strict\";  var { bar } = foo;  bar.hello();}", ecmaFeatures: { destructuring: true } },
```

Comment:
`ecmaFeatures: { destructuring: true }` is not mentioned in https://github.com/eslint/eslint/tree/master/docs/configuring